### ISSUE TITLE
Losing ItemMeta and lore with MiniNBT

### ISSUE THREAD

**Issue by AlessioDP** [stefvanschie/IF#33]
By using MiniNBT it seems that you cannot set ItemMeta and lore anymore, I tried to change the code like this (at line 58~):
```java
NBTWrappers.NBTTagCompound compound = ItemNBTUtil.getTag(item);
compound.setString("IF-uuid", uuid.toString());

this.item = ItemNBTUtil.setNBTTag(compound, item);
```

Now you will not lose ItemMeta data but only lore.

**Comment by stefvanschie** [comment]
I've created an issue on the repository which is used for handling NBT. I'll release an update so that at the very least (some) of the other item properties still work and then we'll have to wait until this other issue is fixed.

**Comment by stefvanschie** [comment]
The update should now be live. Keeping this open until the lore issue is also resolved.

**Comment by stefvanschie** [comment]
The author of MiniNBT has pushed an update which addressed this issue. I'll update this to use that version and then this issue should be resolved completely.

**Comment by stefvanschie** [comment]
Update 0.5.12 should address this issue fully. If this is not solved in version 0.5.12, please let me know and I'll reopen this.
